### PR TITLE
fix issue #930

### PR DESCRIPTION
**Pull request by juise**

### CODE REVIEW

**Comment by thomas-mangin** [comment]
Thank you for the fix .. which would work but it would remove the message. We should get the neighbour from the peer instead - Could I ask you to do so if you do not mind ?
Also add yourself to the CHANGELOG for the work please :-)

**Comment by thomas-mangin** [comment]
If not I will do it a bit later on.

**Comment by juise** [comment]
Yeah, done!

File: lib/exabgp/configuration/neighbor/api.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def _empty (cls):
 	def clear (self):
 		self.api = {}
 		self.named = ''
-		pass
+		Section.clear(self)
```
Comment (thomas-mangin) [inline lib/exabgp/configuration/neighbor/api.py]: Did you find anything wrong that this fixes please ?

Comment (juise) [reply lib/exabgp/configuration/neighbor/api.py]: Yeah, on 'restart', the follow code https://github.com/Exa-Networks/exabgp/blob/2c27112b480132be12a1b3290704f0312769eef9/lib/exabgp/configuration/neighbor/api.py#L147 returns exception/error like

```
19:31:13 | 1342   | configuration | not reloaded, no change found in the configuration
19:31:13 | 1342   | configuration | problem parsing configuration file line 21
19:31:13 | 1342   | configuration | error message: the name "services" already exists in api
```

Comment (juise) [reply lib/exabgp/configuration/neighbor/api.py]: Could you please, create the new build 4.1.3 and upload it into the PyPI? )

**Pull request merged**